backend/src/services/cdm.service.ts

```diff
@@ -336,9 +337,27 @@ export async function optimizeBlockAssignments() {
   }
 }
 
+export async function cleanupUsers() {
+  const usersToBeDeleted = await userModel
+    .find({
+      vacdm: {
+        admin: false,
+        atc: false,
+        banned: false,
+      },
+      updatedAt: {
+        $lte: new Date(Date.now() - config().timeframes.timeSinceLastLogin).getTime(),
+      },
+    })
+    .exec();
+
+  for (let user of usersToBeDeleted) await user.delete();
```

Comment:
please split this line into multiple lines, i.e.

```js
for (const user of users) {
  await user.delete();
}
```

alternatively, it might be better to instead do something like `await Promise.allSettled(usersTobeDeleted.map(user => user.delete()))`, which would allow the deletions to be handled parallelly

Reply:
Of course! Thank you